website/docs/from_provider/quickstart.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Quickstart
+---
+
+## Try Riverpod online
+
+Before anything, we'd suggest you try out Riverpod in a sandbox to test its features out.
+
+The following example should be familiar and readable enough:
+  - dartpad.dev example (link)
+  - marvel app example, or another github repo example (link)
+  - ... more?
+
+## Migrating
+
+This section is designed for people familiar with the [Provider] package who
+wants to learn about Riverpod.
+
+Migrating from Provider to Riverpod can be very straightforward.  
+Migrating basically consists in a few steps that can be done in an *incremental* way.
+
+### Start gently
+*If you have an existing app, don't try to migrate all your providers at once.*  
+Don't immediately try to change your `ChangeNotifiers` into the more modern [AsyncNotifiers] from Riverpod.
+This will requite a bit of a paradigm shift, so it may be difficult to do initially.  
+
+Take your time, as it is important to get yourself familiar with Riverpod first.  
+The following is perfectly fine to start with:
+
+```dart
+// If you have this...
+class MyNotifier extends ChangeNotifier {
+  int state = 0;
+
+  void increment() {
+    state++;
+    notifyListeners();
+  }
+}
+
+// ... just add this!
+final myNotifierProvider = ChangeNotifierProvider<MyNotifier>((ref) {
+  return MyNotifier();
+});
+```
+
+It's fine to keep using `ChangeNotifier` using Riverpod and not use the latest fancy Riverpod features ASAP.
+
+While you should strive toward moving all your application to Riverpod in the long-run, 
+**don't burn yourself out**.  
+Do it one provider at a time.  
+
+### Riverpod and Provider can coexist
+*Keep in mind that it is entirely possible to use both Provider and Riverpod at the same time.*
+
+Indeed, using import aliases, it is possible to use the two APIs altogheter.  
+This is also great for readibilty and it removes any ambiguous API usage.
+
+If you plan on doing this, consider using import aliases for each Provider import in your codebase.
+
+:::info
+A full guide onto how to effectively implement import aliases is incoming soon.
+:::
+
+### Migrate one Provider at a time
+*All Providers from pkg:provider have a strict equivalent in Riverpod*.
+
+In particular, you'll find [ChangeNotifierProvider], which is there to enable migration from Provider.  
+This provider is not recommended when writing new code, and it is not the best way to use Riverpod, but 
+it's a gentle and very easy way to start your migration.
+
+Take the above example. Fully migrating that Provider means writing the following:
+
+```dart
+class MyNotifier extends Notifier<int> {
+  @override
+  int build() => 0;
+
+  void increment() => state++;
+}
+
+final myNotifierProvider = NotifierProvider<MyNotifier, int>(MyNotifier.new);
+```
+
+Thus, migrating consists in moving `provider.Provider` definitions *outside* of `MultiProvider`, 
+defining it as a final global `riverpod.Provider` variable and using `riverpod`'s new classes where needed.
+
+
```

Comment:
Another section that would be valuable to add here would be "Start with _leaves_"

Aka start with providers who do not depend on anything else. So avoid ProxyProviders first basically.
And only start migrating them once all of their dependencies have been migrated.